timers.js

```diff
@@ -1,4 +1,37 @@
+var reset = $("#reset");
+var start = $("#start");
+var pause = $("#pause");
+var timerId;
 
 
+var seconds = 0;
 
+function updateTime(){
+  if (seconds <= 59){
+    seconds++;
+  }
```

Comment:
Totally optional, and sometimes not a good idea, but you can actually write `if` statements without `{}`:

```js
if(seconds <= 59) seconds ++;
else if(something else) console.log("test");
else console.log("foo");
```